### PR TITLE
Net im/toxic: remove version, fix bug and version bump

### PR DESCRIPTION
**Pull request by xWNgLWT0mQf**

### CODE REVIEW

File: net-im/toxic/toxic-0.10.1-r1.ebuild [hunk 1]
```diff
@@ -58,6 +60,8 @@ src_prepare() {
 	#This line changes the "lazy set if absent" assignment to a "lazy set" assignment.
 	#look below in src_configure to see how CFLAGS are passed to the makefile in USER_CFLAGS
 	sed -i -e 's/?=/=/g' Makefile || die "Unable to force CFLAGS and LDFLAGS"
+	#Fix incorrect include declarations for the macros NAME_MAX and PATH_MAX
+	eapply -p0 "${FILESDIR}/${P}-PATH_MAX-and-NAME_MAX.patch" || die "Unable to fix include statements"
```
Comment (juippis) [inline net-im/toxic/toxic-0.10.1-r1.ebuild:64]:
```
 * ERROR: net-im/toxic-0.10.1-r1::gentoo failed (prepare phase):
 *   patch -p1 -p0 failed with /var/tmp/portage/net-im/toxic-0.10.1-r1/files/toxic-0.10.1-PATH_MAX-and-NAME_MAX.patch
 * 
```

**Review by juippis — COMMENTED** [review]
Then I'm getting:
```
[ERROR] >>> Not all runs were successful.
atom: =net-im/toxic-0.11.1, USE flags: 'X audio-notify -games llvm -notification png python -python_single_target_python3_10 python_single_target_python3_8 -python_single_target_python3_9 -qrcode sound -video'
atom: =net-im/toxic-0.11.1, USE flags: 'X audio-notify -games llvm -notification png python -python_single_target_python3_10 python_single_target_python3_8 -python_single_target_python3_9 -qrcode -sound -video'
atom: =net-im/toxic-0.11.1, USE flags: 'X -audio-notify games -llvm notification -png python python_single_target_python3_10 -python_single_target_python3_8 -python_single_target_python3_9 -qrcode sound video'
```
```
>>> Compiling source in /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1 ...
make -j16 -l10 
  CC    autocomplete.o
  CC    avatars.o
  CC    bootstrap.o
  CC    chat.o
  CC    chat_commands.o
  CC    configdir.o
  CC    curl_util.o
  CC    execute.o
  CC    file_transfers.o
  CC    friendlist.o
  CC    global_commands.o
  CC    conference_commands.o
  CC    conference.o
  CC    help.o
  CC    input.o
  CC    line_info.o
  CC    log.o
  CC    message_queue.o
  CC    misc_tools.o
  CC    name_lookup.o
  CC    notify.o
  CC    prompt.o
  CC    qr_code.o
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/qr_code.c:231: warning: ISO C forbids an empty translation unit [-Wpedantic[https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wpedantic]]
  CC    settings.o
  CC    term_mplex.o
  CC    toxic.o
In file included from /usr/include/python3.10/Python.h:8,
                 from /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.h:27,
                 from /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/toxic.c:82:
/usr/include/python3.10/pyconfig.h:1705: warning: "_XOPEN_SOURCE" redefined
 1705 | #define _XOPEN_SOURCE 700
      | 
<command-line>: note: this is the location of the previous definition
  CC    toxic_strings.o
  CC    windows.o
  CC    x11focus.o
  CC    audio_call.o
  CC    audio_device.o
  CC    video_call.o
  CC    video_device.o
  CC    game_base.o
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/game_base.c: In function 'game_new_window':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/game_base.c:882:53: warning: '%s' directive output may be truncated writing up to 127 bytes into a region of size 31 [-Wformat-truncation=[https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-truncation=]]
  882 |         snprintf(ret->name, sizeof(ret->name), "%s (%s)", window_name, nick);
      |                                                     ^~                 ~~~~
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/game_base.c:882:9: note: 'snprintf' output 4 or more bytes (assuming 131) into a destination of size 33
  882 |         snprintf(ret->name, sizeof(ret->name), "%s (%s)", window_name, nick);
      |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  CC    game_centipede.o
  CC    game_chess.o
  CC    game_life.o
  CC    game_util.o
  CC    game_snake.o
  CC    api.o
In file included from /usr/include/python3.10/Python.h:8,
                 from /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.h:27,
                 from /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/api.c:38:
/usr/include/python3.10/pyconfig.h:1705: warning: "_XOPEN_SOURCE" redefined
 1705 | #define _XOPEN_SOURCE 700
      | 
<command-line>: note: this is the location of the previous definition
  CC    python_api.o
In file included from /usr/include/python3.10/Python.h:8,
                 from /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:24:
/usr/include/python3.10/pyconfig.h:1705: warning: "_XOPEN_SOURCE" redefined
 1705 | #define _XOPEN_SOURCE 700
      | 
<command-line>: note: this is the location of the previous definition
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/api.c: In function 'invoke_autoruns':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/api.c:205:67: warning: '%s' directive output may be truncated writing up to 4351 bytes into a region of size 4210 [-Wformat-truncation=[https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wformat-truncation=]]
  205 |                 snprintf(err_buf, sizeof(err_buf), "Invalid path: %s", abspath_buf);
      |                                                                   ^~   ~~~~~~~~~~~
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/api.c:205:17: note: 'snprintf' output between 15 and 4366 bytes into a destination of size 4224
  205 |                 snprintf(err_buf, sizeof(err_buf), "Invalid path: %s", abspath_buf);
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  LD    toxic
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: warning: relocation against `_Py_NoneStruct' in read-only section `.text'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: in function `_Py_DECREF':
/usr/include/python3.10/object.h:500: undefined reference to `_Py_Dealloc'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: in function `python_api_display':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:42: undefined reference to `PyArg_ParseTuple'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:47: undefined reference to `_Py_NoneStruct'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: in function `python_api_get_nick':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:55: undefined reference to `PyArg_ParseTuple'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:65: undefined reference to `Py_BuildValue'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: in function `python_api_get_status':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:74: undefined reference to `PyArg_ParseTuple'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:80: undefined reference to `Py_BuildValue'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:84: undefined reference to `Py_BuildValue'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:88: undefined reference to `Py_BuildValue'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: in function `python_api_get_status_message':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:100: undefined reference to `PyArg_ParseTuple'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:110: undefined reference to `Py_BuildValue'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: in function `python_api_get_all_friends':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:120: undefined reference to `PyArg_ParseTuple'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:125: undefined reference to `PyList_New'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:133: undefined reference to `Py_BuildValue'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:134: undefined reference to `PyList_Append'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: in function `python_api_send':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:144: undefined reference to `PyArg_ParseTuple'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:149: undefined reference to `_Py_NoneStruct'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: in function `python_api_execute':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:157: undefined reference to `PyArg_ParseTuple'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:162: undefined reference to `_Py_NoneStruct'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: in function `python_api_register':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:172: undefined reference to `PyArg_ParseTuple'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:176: undefined reference to `PyCallable_Check'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:177: undefined reference to `PyExc_TypeError'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:177: undefined reference to `PyErr_SetString'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:182: undefined reference to `PyExc_TypeError'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:182: undefined reference to `PyErr_SetString'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:199: undefined reference to `PyErr_NoMemory'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:206: undefined reference to `PyErr_NoMemory'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:214: undefined reference to `PyErr_NoMemory'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:224: undefined reference to `_Py_NoneStruct'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:225: undefined reference to `_Py_NoneStruct'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: in function `PyInit_toxic_api':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:250: undefined reference to `PyModule_Create2'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:251: undefined reference to `Py_BuildValue'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:252: undefined reference to `Py_BuildValue'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:253: undefined reference to `Py_BuildValue'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:254: undefined reference to `PyObject_SetAttrString'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:255: undefined reference to `PyObject_SetAttrString'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:256: undefined reference to `PyObject_SetAttrString'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: in function `terminate_python':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:278: undefined reference to `Py_Finalize'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: in function `init_python':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:284: undefined reference to `PyImport_AppendInittab'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:285: undefined reference to `Py_Initialize'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: in function `run_python':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:290: undefined reference to `PyRun_SimpleFileExFlags'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/python_api.o: in function `do_python_command':
/var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:305: undefined reference to `PyList_New'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:308: undefined reference to `Py_BuildValue'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:308: undefined reference to `PyList_Append'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:311: undefined reference to `PyTuple_Pack'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/src/python_api.c:313: undefined reference to `PyObject_CallObject'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.2.0/../../../../x86_64-pc-linux-gnu/bin/ld: warning: creating DT_TEXTREL in a PIE
collect2: error: ld returned 1 exit status
make: *** [Makefile:76: /var/tmp/portage/net-im/toxic-0.11.1/work/toxic-0.11.1/build/toxic] Error 1
 * ERROR: net-im/toxic-0.11.1::gentoo failed (compile phase):
 *   emake failed
```
It seems to related to `+python` USE flag. Any ideas?

File: net-im/toxic/toxic-0.10.1-r1.ebuild (hunk 1 above)
Comment (xWNgLWT0mQf) [reply net-im/toxic/toxic-0.10.1-r1.ebuild]: I am unable to replicate the error regarding the patch for toxic-0.10.1-r1. I was able to replicate and fix (I hope) the linking with python error.

**Review by juippis — APPROVED** [review]
0.8 still fails with python, but 0.11.1 seems to be working now. I'll add the BDEPEND, thanks!

**Pull request closed without merging**